doc/tutorial/get_started.rst

```diff
@@ -145,6 +145,17 @@ Complete the following steps to set up this network:
      lxc network get microbr0 ipv4.address
      lxc network get microbr0 ipv6.address
 
+You also have the option to create a dedicated network for the Ceph traffic:
```

Comment:
This is a tutorial, so we shouldn't offer choices but decide whether this is something we want the reader to do or not.

Is it a rather special use case, or should anybody who tries out MicroCloud for the first time do this?

(An alternative could be to add a second tutorial for this scenario.)

Reply:
fixed